test/MainEvent.cxx

```diff
@@ -174,6 +186,7 @@ int main(int argc, char **argv)
       Int_t nentries = (Int_t)tree->GetEntries();
       nevent = TMath::Min(nevent,nentries);
       if (read == 1) {  //read sequential
+         ioperf = getenv("ENABLE_TTREEPERFSTATS") ? new TTreePerfStats("Perf Stats", tree) : nullptr;
```

Comment:
My bad, I had missed it ... but now I also took another look at the commit messages ... for the sake of our future selves can you update the commit message that just say things like "Small fixups from review." into a short description of the functional changes :).    Thanks.